CHANGELOG.md

```diff
@@ -6,3 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+### Added
+
+- ComponentState helper in tests (#836)
```

Comment:
```suggestion
### Changed

- Use ComponentState in tests (#836)
```
isn't this more accurate? it's not really a helper, and it's a change rather than an addition